lib/config/contributors.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const YAML = require('yaml').default;
+const fs = require('../deps/fsp.js');
+const path = require('path');
+
+const CONTRIBUTORS_CONFIG_LOCATION = path.resolve(__dirname, '..', '..', 'contributors', '_contributors.yaml');
```

Comment:
Do you need to separate all of these or is it possible to do:

```js
path.resolve(__dirname, '../../contributors/_contributors.yaml')
```

Reply:
That would not work on Windows. This is the OS cross-compatible way of doing directory traversal.

Reply:
ah good point!